Source/CesiumRuntime/Private/Cesium3DTilesetRoot.cpp

```diff
@@ -82,7 +82,7 @@ void UCesium3DTilesetRoot::_updateAbsoluteLocation() {
 void UCesium3DTilesetRoot::_updateTilesetToUnrealRelativeWorldTransform() {
   ACesium3DTileset* pTileset = this->GetOwner<ACesium3DTileset>();
 
-  const glm::dmat4& ellipsoidCenteredToUnrealWorld =
+  const glm::dmat4 ellipsoidCenteredToUnrealWorld =
```

Comment:
For some reason, on MacOS, the reference becomes invalid when its used at the end of the function.

Reply:
And I assume it crashes? Can you post a callstack. I'm curious.

Reply:
And I assume it crashes? 

Can I reproduce it? (or at least run the same level you are? ) I'm a little curious to see what's going on.

Reply:
It doesnt crash, however it causes the tiles to not show up. That's because it will cause the camera location to be somewhere around the center of the earth, with the matrix being cleared of its ECEF coordinates.

I can only reproduce this bug in Mac. I guess their compilers/c++ runtime disposes of temporary variables sooner.

Reply:
Ok thanks for that. Here's some more explanation from @kring 
> The returned GeoTransforms is tenporary in V2.0. So the matrix is as well. That method should not be using GeoTransforms at all anymore - it's obsolete. Not sure how that one got by me. So go ahead and get rid of the ampersand for now, that's a good fix to start with